BSB_lan.ino

```diff
@@ -470,12 +470,14 @@ const char *journalFileName = "journal.txt";
 
 #ifdef WIFI
 WiFiEspClient client;
+WiFiEspClient mqtt_client;  //Luposoft: own instance 
 #ifdef VERSION_CHECK
 WiFiEspClient httpclient;
 #endif
 WiFiEspClient telnetClient;
 #else
 EthernetClient client;
+EthernetClient mqtt_client;   //Luposoft: own instance
```

Comment:
when we use client variable, there is not a stable connection to mqtt.
I don't have a final explanation for it, I come to this wisdom while try and error...
But in my opinion 'client.flush' is disturbing the mqtt-connection

Reply:
Here was mistake in my mind. You right. For persistent connection it will be better.
Note:
 EthernetClient *mqtt_client; //by reference
would be better than 
EthernetClient mqtt_client; //by value
because when MQTT is not used mqtt_client consume lesser RAM.

Reply:
This was a long journey till I find out, *mqtt_client won't work.
With the * it's also possible to compile und upload, but the program will frozen at the last serial output before mqtt-statement. (like mqtt->loop)